jbpm-services/jbpm-executor/src/test/java/org/jbpm/executor/impl/wih/AsyncContinuationSupportTest.java

```diff
@@ -1208,7 +1208,7 @@ public void testAsyncModeWithSignalEventSubProcess() throws Exception {
     @Test(timeout = 10000)
     public void testAsyncModeWithInclusiveGateway() throws Exception {
         // JBPM-7414 // there are two paths for end process , therefore it is executed twice
-        final NodeLeftCountDownProcessEventListener countDownListener = new NodeLeftCountDownProcessEventListener("EndProcess", 2);
+        final NodeLeftCountDownProcessEventListener countDownListener = new NodeLeftCountDownProcessEventListener("EndProcess", 1);
```

Comment:
changing this was incorrect. see this 
https://github.com/kiegroup/jbpm/pull/1270/files

Reply:
@elguardian hmm, looks obvious now :) Thanks for having a look at it.